runbotics/runbotics-orchestrator-ui/.eslintrc.js

```diff
@@ -1,148 +1,94 @@
-                    "JSXNamespacedName",
-                    "JSXMemberExpression",
-                    "JSXSpreadAttribute",
-                    "JSXExpressionContainer",
-                    "JSXOpeningElement",
-                    "JSXClosingElement",
-                    "JSXFragment",
-                    "JSXOpeningFragment",
-                    "JSXClosingFragment",
-                    "JSXText",
-                    "JSXEmptyExpression",
-                    "JSXSpreadChild",
-                ],
-                ignoreComments: false,
-            },
-        ],
-        // Same as eslint-config-airbnb-base excluding for..of loop
-        "no-restricted-syntax": [
-            "error",
-            {
-                selector: "ForInStatement",
-                message:
-                    "for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.",
-            },
-            {
-                selector: "LabeledStatement",
-                message:
-                    "Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.",
-            },
-            {
-                selector: "WithStatement",
-                message:
-                    "`with` is disallowed in strict mode because it makes code impossible to predict and optimize.",
-            },
-        ],
-        "react/destructuring-assignment": "off",
-        "react/jsx-filename-extension": "off",
-        "react/prop-types": "off",
-        "react/require-default-props": "off",
-        "react/jsx-indent": "off",
-        "react/jsx-indent-props": "off",
-        "react/jsx-props-no-spreading": "off",
-        "react-hooks/exhaustive-deps": "off",
-    },
-        settings: {
-        react: {
-            version: "detect", // Tells eslint-plugin-react to automatically detect the version of React to use
-        },
-                "import/resolver": {
-            node: {
-                extensions: [".js", ".jsx", ".ts", ".tsx"],
-            },
-        },
-    },
-    overrides: [
-        {
-            files: ["*.styled.ts", "*.styles.ts"],
-            rules: {
-                "no-magic-numbers": "off",
-                "import/prefer-default-export": "off",
-            },
-        },
-                {
-            files: ["*.js"],
-            rules: {
-                // Allow `require()`
-                "@typescript-eslint/no-var-requires": "off",
-            },
-        },
-    ],
-        ignorePatterns: ["**/node_modules/**", "**/build/**", "**/dist/**"],
-}
+module.exports = {
+    extends: 'next/core-web-vitals',
+    plugins: ['unused-imports', '@typescript-eslint', 'simple-import-sort'],
+    ignorePatterns: ['**/*.js'],
+    rules: {
+        indent: ['error', 4, { SwitchCase: 1 }],
+        'comma-dangle': ['warn', 'only-multiline'],
+        'react-hooks/rules-of-hooks': 'error',

```

Comment:
disable this one `react-hooks/exhaustive-deps`

Reply:
Not sure if its good idea (source https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528)

Reply:
from my point of view when I know what should be in the deps it is annoying because we don't want to include every function and so one in deps. It's better for newbies but we can cover it via CR and tests on dev